### PR TITLE
updated public sysdig end points

### PR DESCRIPTION
**Pull request by SaiKiranAre**
updated the public sysdig endpoints using this url: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints

And also us-south and us-east added the br-sao and ca-tor sysdig endpoints for single region deployment usecases

### CODE REVIEW

File: calico-policies/public-network-isolation/us-east/allow-public-services-pods.yaml [hunk 1]
```diff
@@ -37,16 +37,15 @@ spec:
...
+      # IBM Cloud Monitoring with Sysdig: us-east
+      # Taken from: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints
       - 169.60.112.74/32
-      - 169.61.109.59/32
-      - 169.61.65.234/32
+      - 169.55.109.114/32
       - 169.62.3.82/32
-      - 169.63.158.254/32
-      - 169.63.163.50/32
+      # IBM Cloud Monitoring with Sysdig: ca-tor subnets
```
Comment (bradbehle) [inline calico-policies/public-network-isolation/us-east/allow-public-services-pods.yaml:45]: Why do us-east clusters need to access ca-tor sysdig IPs?  I don't see this mentioned in the docs specified here: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints

File: calico-policies/public-network-isolation/us-south/allow-public-services-pods.yaml [hunk 2]
```diff
@@ -37,14 +37,15 @@ spec:
...
+      - 169.60.151.174/32
       - 169.46.0.70/32
-      - 169.47.102.30/32
-      - 169.47.227.243/32
       - 169.48.214.70/32
-      - 169.60.151.174/32
-      - 169.62.255.114/32
-      - 169.62.255.116/32
+      # IBM Cloud Monitoring with Sysdig: br-sao subnets
+      - 163.107.66.98/32
```
Comment (bradbehle) [inline calico-policies/public-network-isolation/us-south/allow-public-services-pods.yaml:46]: Why do us-south clusters need to access br-sao sysdig IPs? I don't see this mentioned in the docs specified here: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints

**Review by bradbehle — CHANGES_REQUESTED** [review]
One more overall question, can you confirm that removing some of these old sysdig/monitoring IPs will not break any monitoring function on any old existing IKS/ROKS clusters, including clusters where sysdig or the cluster workers haven't been updated in a while?

File: calico-policies/public-network-isolation/us-east/allow-public-services-pods.yaml (hunk 1 above)
Comment (SaiKiranAre) [reply calico-policies/public-network-isolation/us-east/allow-public-services-pods.yaml:45]: @bradbehle , when deploying to single zone ca-tor, we are using us-east policies as mentioned in the note. That is the reason

```
NOTE: The policies in the ca-tor and br-sao directories are meant for use with the Toronto and São Paulo multizone locations. For the Toronto single zone location, use the policies in the us-east directory instead. For the São Paulo single zone location, use the policies in the us-south directory instead.
```

File: calico-policies/public-network-isolation/us-south/allow-public-services-pods.yaml (hunk 2 above)
Comment (SaiKiranAre) [reply calico-policies/public-network-isolation/us-south/allow-public-services-pods.yaml:46]: @bradbehle , when deploying to single zone br-sao, we are using us-south policies as mentioned in the note. That is the reason

**Comment by SaiKiranAre** [comment]
> One more overall question, can you confirm that removing some of these old sysdig/monitoring IPs will not break any monitoring function on any old existing IKS/ROKS clusters, including clusters where sysdig or the cluster workers haven't been updated in a while?

@bradbehle , let me check with sysdig team if there is any host which points to the old ips. Will update this accordingly. Thanks a lot

File: calico-policies/public-network-isolation/us-south/allow-public-services-pods.yaml (hunk 2 above)
Comment (bradbehle) [reply calico-policies/public-network-isolation/us-south/allow-public-services-pods.yaml:46]: Oh, I forgot about those single zone regions.  Thanks

**Comment by SaiKiranAre** [comment]
@bradbehle added the old ips as they may be used as backup as discussed here:
https://ibm-cloudplatform.slack.com/archives/GDGB25SP6/p1672899911546279

**Review by bradbehle — APPROVED** [review]

**Review by derekpoindexter — APPROVED** [review]

**Comment by IdanAdar** [comment]
@SaiKiranAre Is this getting merged?

**Comment by SaiKiranAre** [comment]
> @SaiKiranAre Is this getting merged?

@bradbehle , Is there any process to merge the PR?

**Comment by mihivagyok** [comment]
As it has the required approves, I merge it. Thanks!

**Pull request merged**